web/cookiesCTF/main.py

```diff
@@ -0,0 +1,122 @@
+from bottle import route, run, request, response, static_file, get, template
+
+
+@route('/')
+def hello():
+	f = open('./Pages/index.html')
+	return f.read()
+
+
+@get('/styles.css')
+def retCss():
+	return static_file('styles.css', root='./Pages')
+
+
+@get('/', method='POST')
+def checkFlag():
+	flag = request.forms.get('flag')
+	if flag == "PB{eat_my_cookies}" or flag == "PB{I'm_a_cookiebot}" or flag == "PB{Jake_don't_mind}" or flag == "PB{Jake_isn't_lonely}" or flag == 'PB{Y0tub3_v1d30s_ar3_fun}':
```

Comment:
Instead of making many separate checks let's create somewhere a [set](https://docs.python.org/3/tutorial/datastructures.html#sets) of correct flags, and here just check if `flag` is `in` that set